modules/statics/src/networks.ts

```diff
@@ -228,6 +228,19 @@ class DashTestnet extends BitcoinLikeTestnet {
   family = CoinFamily.DASH;
   explorerUrl = 'https://testnet-insight.dashevo.org/insight/tx/';
 }
+
+class Polkadot extends Mainnet implements AccountNetwork {
+  name = 'Polkadot';
+  family = CoinFamily.DOT;
+  explorerUrl = 'https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/explorer';
```

Comment:
This is the base URL to use to find a transaction. `https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/explorer` cannot be used to build a tx URL